discovery-provider/src/queries/get_health.py

```diff
@@ -250,6 +258,36 @@ def get_health(args: GetHealthArgs, use_redis_cache: bool = True) -> Tuple[Dict,
         )
         last_track_unavailability_job_end_time = None
 
+    try:
+        redis_result = redis.get(index_user_bank_backfill_complete)
+        if redis_result:
+            is_index_user_bank_backfill_complete = str(redis_result.decode())
+    except Exception as e:
+        logger.error(
+            f"Could not check whether index_user_bank backfilling is complete: {e}"
+        )
+        is_index_user_bank_backfill_complete = None
+
+    try:
+        redis_result = redis.get(index_rewards_manager_backfill_complete)
+        if redis_result:
+            is_index_rewards_manager_backfill_complete = str(redis_result.decode())
+    except Exception as e:
+        logger.error(
+            f"Could not check whether index_rewards_manager backfilling is complete: {e}"
+        )
+        is_index_rewards_manager_backfill_complete = None
+
+    try:
+        redis_result = redis.get(index_spl_token_backfill_complete)
+        if redis_result:
+            is_spl_token_backfill_complete = str(redis_result.decode())
```

Comment:
nit: maybe bool?

Reply:
Correct me if I'm wrong but I don't think we can store bools in redis, so I'm storing a string that says either 'false' or 'true' and just checking against those values.